src/borg/archive.py

```diff
@@ -569,7 +569,7 @@ def extract_item(self, item, restore_attrs=True, dry_run=False, stdout=False, sp
         path = os.path.join(dest, item.path)
         # Attempt to remove existing files, ignore errors on failure
         try:
-            st = os.lstat(path)
+            st = os.stat(path, follow_symlinks=False)
```

Comment:
2nd and 3rd rather cleanups (== always use generic call with follow_symlinks=False).

consistency. couldn't get all switched though due to lchmod on linux.